index.html

```diff
@@ -6,59 +6,244 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <title>Shingirai Portfolio</title>
+    
 </head>
 
 <body>
     <header>
-        <a href="/" class="header-logo">
-            <h3>
-                Welcome
-            </h3>
-        </a>
-        <nav>
-            <img src="./assets/Union.svg" alt="menu link">
-        </nav>
+        <div class="header-nav-phone">
+            <a href="/" class="header-logo">
+                <h3>
+                    Welcome
+                </h3>
+            </a>
+            <nav>
+                <img src="./assets/Union.svg" alt="menu link">
+            </nav>
+        </div>
     </header>
     <section class="intro">
-        <h1>Hey there. I’m Shingirai<span><br>
-            I’m a software developer</span></h1>
-        <p>
-            I can help you build a product , feature or website Look through some of my work and experience! If you like
-            what you see and have a project you need coded,
-            don’t hestiate to contact me.
-        </p>
+        <nav>
+            <ul class="header-nav-desktop">
+                <li>Portfolio</li>
```

Comment:
- [ ] Please use grid layout to align the section that contain the works projects
![image](https://user-images.githubusercontent.com/74744787/159077427-ca832f62-d1b6-4187-8ec0-c0e8da321d31.png)
![image](https://user-images.githubusercontent.com/74744787/159077476-0363eeae-1ff0-47b4-9638-16c45056e5f0.png)

you can easily notice that the cards don't have the proper position